CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 - Don't include changes that are purely internal. The CHANGELOG should be a
   useful summary for people upgrading their application, not a replication
   of the commit log.
+  
+## Unreleased
+
+* Add GovernmentService schema.org schema ([PR #1177](https://github.com/alphagov/govuk_publishing_components/pull/1177))
```

Comment:
Can we be more specific?
```suggestion
* Add GovernmentService schema.org schema to machine readable metadata component([PR #1177](https://github.com/alphagov/govuk_publishing_components/pull/1177))
```

Reply:
I was following the prior art of previous commits here - I'm happy with either.

Reply:
The changelog should match the PR title. If more detail is needed they should both be updated, but hopefully the description gives enough information.

Reply:
Yup - for me I found the title wasn't enough to understand what's happened even and reads as an incomplete sentence. (Sorry its a minor thing)